bims/api_views/location_site.py

```diff
@@ -289,6 +294,138 @@ def get(self, request):
             return Response(response_data)
 
 
+    def get_biodiversity_data(self, collections):
+        biodiversity_data = {}
+        taxa = self.get_origin_cons_endemsim_data(collections)
+        # If I found more than one class of animal
+        if 'Actinopterygii' not in taxa:
+            print('Hey there, you are not a fish!')
+        else:
+            biodiversity_data['fish'] = {}
+            biodiversity_data['fish']['origin_chart'] = {}
+            biodiversity_data['fish']['cons_status_chart'] = {}
+            biodiversity_data['fish']['endemism_chart'] = {}
+            biodiversity_data['fish']['origin_chart']['data'] = (
+                taxa['Actinopterygii']['origin_chart']['data'])
```

Comment:
I dont understand why you have hard coded Actinopterygii in everywhere. This should be modelled in data, not hard coded in logic....Check with @dimasciput that we have grouping entities in the data model for fish, algae and inverts and use that to join onto the taxon table the relevant records.

Reply:
Yes we have Taxon group model in https://github.com/kartoza/django-bims/blob/develop/bims/models/taxon_group.py. And you can check the category from here : 
https://github.com/kartoza/django-bims/blob/develop/bims/enums/taxonomic_group_category.py.

Reply:
Great, will fix this 👍